src/activate/registerCommands.ts

```diff
@@ -310,3 +303,84 @@ export const openClineInNewTab = async ({ context, outputChannel }: Omit<Registe
 
 	return tabProvider
 }
+
+export const openSettingsInNewTab = async ({ context, outputChannel }: Omit<RegisterCommandOptions, "provider">) => {
+	const contextProxy = await ContextProxy.getInstance(context)
+
+	// Get the existing MDM service instance to ensure consistent policy enforcement
+	let mdmService: MdmService | undefined
+	try {
+		mdmService = MdmService.getInstance()
+	} catch (error) {
+		// MDM service not initialized, which is fine - extension can work without it
+		mdmService = undefined
+	}
+
+	const tabProvider = new ClineProvider(context, outputChannel, "editor", contextProxy, mdmService)
+	const lastCol = Math.max(...vscode.window.visibleTextEditors.map((editor) => editor.viewColumn || 0))
+
+	// Check if there are any visible text editors, otherwise open a new group
+	// to the right.
+	const hasVisibleEditors = vscode.window.visibleTextEditors.length > 0
+
+	if (!hasVisibleEditors) {
+		await vscode.commands.executeCommand("workbench.action.newGroupRight")
+	}
+
+	const targetCol = hasVisibleEditors ? Math.max(lastCol + 1, 1) : vscode.ViewColumn.Two
+
+	// Get the localized title for the settings tab
+	const settingsTitle = t("settings:tabTitle") || "Roo Code Settings"
+
+	const newPanel = vscode.window.createWebviewPanel(
+		"rooCodeSettings", // Unique ID for settings panel
+		settingsTitle,
+		targetCol,
+		{
+			enableScripts: true,
+			retainContextWhenHidden: true,
+			localResourceRoots: [context.extensionUri],
+		},
+	)
+
+	// Save as tab type panel.
+	setPanel(newPanel, "tab")
+
+	// Use the same icon as the main panel
+	newPanel.iconPath = {
+		light: vscode.Uri.joinPath(context.extensionUri, "assets", "icons", "panel_light.png"),
+		dark: vscode.Uri.joinPath(context.extensionUri, "assets", "icons", "panel_dark.png"),
+	}
+
+	await tabProvider.resolveWebviewView(newPanel)
+
+	// Immediately send a message to open the settings view
+	newPanel.webview.postMessage({ type: "action", action: "settingsButtonClicked" })
```

Comment:
Send didBecomeVisible immediately to mirror the old behavior where both messages were posted to ensure reliable initial scroll/visibility handling.

```suggestion
newPanel.webview.postMessage({ type: "action", action: "settingsButtonClicked" })
newPanel.webview.postMessage({ type: "action", action: "didBecomeVisible" })
```